beancount_reds_importers/libreader/csvreader.py

```diff
@@ -67,14 +67,15 @@ def initialize_reader(self, file):
         if getattr(self, "file", None) != file:
             self.file = file
             self.reader_ready = self.deep_identify(file)
+            self.file_encoding = getattr(self, "file_encoding", None)
```

Comment:
This needs to be one line above, since it needs to be set before `deep_identify` is called. Actually, better yet, it's best to just remove this line, since we then have to depend on `initialize_reader` running. See below.

Reply:
Agree! Updating.